.coafile

```diff
@@ -29,6 +29,8 @@ bears = SpaceConsistencyBear
 
 [commit]
 bears = GitCommitBear
+host_used = github
+contains_url = True
```

Comment:
Do I combine url presence with the presence of git keywords as suggested [here](https://help.github.com/articles/closing-issues-via-commit-messages/)? @jayvdb